content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -173,6 +186,23 @@ span link connecting it to the producer trace.
 You can disable the agent using `-Dotel.javaagent.enabled=false` (or using the
 equivalent environment variable `OTEL_JAVAAGENT_ENABLED=false`).
 
+### Enable only specific instrumentation
+
+You can disable all default auto instrumentation and selectively re-enable individual instrumentation.
+This may be desireable to reduce startup overhead or to have more control of which instrumentation is applied.
+
+- Disable all instrumentation in the agent using `-Dotel.instrumentation.common.default-enabled=false`
+(or using the equivalent environment variable `OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED=false`).
+- Enable each desired instrumentation individually using `-Dotel.instrumentation.[name].enabled=true`
```

Comment:
Would it make sense to add a warning here that sometimes instrumentations depend on other instrumentations. It is up to the people who use this flag to discover such dependencies and make sure that the instrumentation works as expected.

Reply:
Added.